stories/layout/Box.stories.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { storiesOf, addDecorator } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+
+import { Box } from '@sparkpost/matchbox/components/Box';
+import { ThemeProvider } from '@sparkpost/matchbox/components/ThemeProvider';
+
+addDecorator((storyFn) => <ThemeProvider>{storyFn()}</ThemeProvider>);
+
+export default {
+  title: 'Layout|Box',
+};
+
+export const StyledBox = withInfo()(() => (
+  <Box
+    color="gray.1000"
```

Comment:
Is there any way we could have exported constants to use instead of having to depend on all of these string props? Feels like it's pretty susceptible to typos and we'd get free IDE integration for autocomplete. Haven't messed with styled components before, so I'm not sure how much of a pain it would be.

Reply:
We can always import tokens directly from `@sparkpost/design-tokens`, but I think it will be fine once we learn token names and when documentation is set up. 

We could in the future explore adding custom proptype definitions or even typescript to help a bit, but one of the benefits of using styled-system is the flexibility from using strings. eg: `color='#000000'` or `width='100px'`, which we wouldn't be able to check for